optimize/util/optimize-commons/src/test/java/io/camunda/optimize/upgrade/os/OpenSearchClientBuilderTest.java

```diff
@@ -93,7 +93,7 @@ void buildExtendedClient_HappyPath() throws Exception {
     }
 
     // then
-    final var reqWrapper = (BasicHttpRequest) context.getAttribute("http.request");
+    final var reqWrapper = (BasicHttpRequest) context.getRequest();
```

Comment:
Does it need casting still?

Reply:
Lol, no 😆.

I'll rebase after all tests pass here to avoid spending time on build